test/integration/Main.hs

```diff
@@ -28,7 +28,7 @@ main = do
     hspec $ do
         describe "Cardano.WalletSpec" Wallet.spec
         describe "Cardano.Wallet.Network.HttpBridge" HttpBridge.spec
-        beforeAll startCluster $ afterAll killCluster $ do
+        beforeAll startCluster $ afterAll killCluster $ after deleteAllWallets $ do
```

Comment:
Does `after deleteAllWallets` runs after every `it` within each `describe` below, or, does it run after each `describe` only :thinking: ? (we do want the former)
What if we have nested `describe` ?

Reply:
Yes, it works after every `it`, even when they are nested in the subsequent `describes`, which is very nice :+1: